examples/sglang/components/frontend.py

```
@@ -14,20 +14,27 @@
 
 import logging
+import os
 import subprocess
 from pathlib import Path
+from typing import AsyncIterator
 
 from components.worker import SGLangWorker
-from fastapi import FastAPI
+from components.kv_router import Router
 from pydantic import BaseModel
+from utils.protocol import Tokens, PreprocessedRequest, StopConditions, SamplingOptions
```

Ruff (0.11.9)

`utils.protocol.Tokens` imported but unused

Remove unused import

(F401)